packages/playground/define/__tests__/define.spec.ts

```diff
@@ -20,4 +20,6 @@ test('string', async () => {
   expect(await page.textContent('.spread-array')).toBe(
     JSON.stringify([...defines.__STRING__])
   )
+  // html would't need to define replacement
+  expect(await page.textContent('.exp-define')).toBe('__EXP__')
```

Comment:
We should test it in production mode, or in both development and production modes.

Reply:
they have the same expect?

Reply:
see my latest comment